webproject/src/main/webapp/js/room-my-order.js

```diff
@@ -9,7 +9,7 @@ window.onload = function() {
 async function getMyOrder() {
     let orderResponse = await fetch(`/myOrder?roomId=${roomID}`);
     let myOrders = await orderResponse.json();
-    let myOrderItems = Object.values(myOrders);
+    let myOrderItems = Object.values(myOrders)[0];
```

Comment:
what does this for?

Reply:
please use myOrders.orders

Reply:
Thank you, I didn't realise I could do that! I've changed it.